docs/docs/users/general/creating-campaigns.md

```diff
@@ -0,0 +1,42 @@
+---
+id: creating-campaigns
+title: Creating and managing campaigns
+---
+Here is more information about campaigns. These can be either petition campaigns, event campaigns or fundraising campaigns. 
+
+## General info
+![](https://s3-eu-west-1.amazonaws.com/bureaubolster/Wings_2019-03-25_11-31-17.png)
+
+At Campaigns, you can manage 1) Events or 2) Petitions. You can 3) click on existing campaign to edit it, or 4) Create a new one. 
+
+Events are geared towards getting people to visit an event; petitions are aimed towards letting people sign a policy proposal, letter of support or some other kind of petition. 
+
+## Where is your campaign page?
+Campaigns get their own page on your website by default; by defining the 'slug', you define its url.  Every type of campaign has its own directory, though; if your slug is 'demo-fundraiser' and your website url is mywebsite.com, the url of this specific fundraiser page can be found at 'mywebsite.com/fundraisers/demo-fundraiser'. 
+
+![](https://bolster-files.s3-eu-west-1.amazonaws.com/campaignslug2.png)
+
+
+## 'General'
+
+![](https://s3-eu-west-1.amazonaws.com/bureaubolster/Wings_2019-03-25_11-36-27.png)
+
+1. The 'General' tab is what you see on the screenshot above. Here, you can edit the general copy of the petition page. 
+2. At 'Platforms', you can configure the settings for how the website shows up in search engines and on social media.
```

Comment:
This captures the essence a bit better I think:

At platforms, you can configure metadata related to how your content should show up on other platforms, like Facebook or Twitter.